### ISSUE TITLE
sensor translate

### ISSUE THREAD

**Issue by riddik14** [PiotrMachowski/Home-Assistant-custom-components-Burze.dzis.net#28]
hi the package returns the sensors only in polish language, is it possible to translate them with an english file? Thank you

**Comment by PiotrMachowski** [comment]
Yes, it should be doable, but it unfortunately it will make some breaking changes
